extensions/extensions-helm/extensions-helm-pipeline-invocation/aissemble-pipeline-invocation-app-chart/templates/rbac.yaml

```diff
@@ -1,24 +1,28 @@
 {{- /*
 aiSSEMBLE Custom rbac.yaml
 
-Required custom rbac.yaml file that grants the sparkoperator service account 
+Required custom rbac.yaml file that grants the pipeline-invocation-service service account 
 create, delete, and update access to the apigroup apiextensions.k8s.io. 
 
 This is necessary for the pipeline-invocation-service to create instances of the 
 SparkApplication CRD to submit pipelines to the Spark Operator for execution.
 */}}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ .Values.rbac.serviceAccountName | default "pipeline-invocation-service" }}
```

Comment:
I: `serviceaccount` will always be created but the CR, CRB can be turn off if it's not needed. This can avoid the missing service account issue from the `deployment` resource when the `rbac.createClusterRole` is set to `false`

Reply:
A: This makes me feel like they should live in separate YAML files. I don't feel super strongly, but it seems like if the Service Account isn't tightly coupled to the role/role binding then they shouldn't share a file.